scripts/dataset_processing/tts/add_normalized_text.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
I will have a more streamlined script for this soon as part of the new preprocessing pipeline.